install_sct

```diff
@@ -693,8 +693,14 @@ if [[ $OS == linux ]]; then
   conda install -y -c conda-forge libstdcxx-ng
   # Ensure that libffi isn't linked incorrectly (https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/3927#issuecomment-1573896770)
   conda install -y -c conda-forge libffi
-  # Ensure that PyQt5 has the correct dependencies on Ubuntu
+  # Ensure that PyQt5 has the correct dependencies on Ubuntu (https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/3925)
+  # NB: We also need to copy the necessary dependencies to a separate subfolder that we add to LD_LIBRARY_PATH in compat/launcher.py
+  #     We do this to avoid exposing the full `lib/` directory, which can cause conflicts with system libraries.
+  #     This should copy 1 .so file and 2 symlinks, but the symlinks are relative, so copying them to a separate folder should be OK.
+  #     Related issue: https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/4971
   conda install -y -c conda-forge xorg-libxinerama
+  mkdir "$SCT_DIR/$PYTHON_DIR/envs/venv_sct/sct_ld_library_path/"
+  find "$SCT_DIR/$PYTHON_DIR/envs/venv_sct/lib/" | grep -i "libxcb-xinerama.so.*" | xargs -I {} cp {} "$SCT_DIR/$PYTHON_DIR/envs/venv_sct/sct_ld_library_path/"
```

Comment:
Since the `lib/` directory is just a flat list of files and symlinks, I guess `find` and `grep` are overkill. How about:
```suggestion
  cp "$SCT_DIR/$PYTHON_DIR/envs/venv_sct/lib/libxcb-xinerama.so."* "$SCT_DIR/$PYTHON_DIR/envs/venv_sct/sct_ld_library_path/"
```